entry.sh

```diff
@@ -18,5 +18,5 @@ fi
 
 cd build 
 cmake ..
-make
+make -j4
```

Comment:
just `-j` or do some experimenting to find the fastest number based on `nprocs` (I've heard `nprocs`/`nprocs+1`/`nprocs*2` are all potentials)

Reply:
cool will investigate, `-j4` was the highest I could go without it failing